Debug warnings

DarrenWestwood opened the pull request:
Fixes for Debug Mode Warnings #140

blockonomics commented:
@jusasiiv please also check if checkout and saving settings is working properly after these changes in debug mode

DarrenWestwood commented:
> Test Setup is not setting callback URL automatically anymore

Fixed setting callback URL automatically:
https://github.com/blockonomics/woocommerce-plugin/pull/142/commits/66896694b4d8a6f0f63eb5f8290df626682d5830#diff-19af2d132794caa989960435bf4ad733R167 

> Also these nested ternaries are not very readable

Fixed readability
